tests/hopp/test_offshore/test_offshore.py

```diff
@@ -5,6 +5,10 @@
 from ORBIT import load_config
 from hopp.offshore.fixed_platform import install_platform, calc_platform_opex, calc_substructure_mass_and_cost
 
+'''Sources:
+    - [1] 2017 ORBIT Technical Report: https://www.nrel.gov/docs/fy17osti/66874.pdf
```

Comment:
Can you just add in the authors, title, etc for the citation? Otherwise I think we are ready to merge.